### ISSUE TITLE
Split up into different repos

### ISSUE THREAD

**Issue by paulpach** [pernyblom/pernyblom.github.io#1]
This is a suggestion,  each one of these projects should have their own github repository.

**Comment by pernyblom** [comment]
Yes, they should. Currently this is just a repo for hosting my projects, so it would probably be good to split them.

/Per

________________________________
Från: Paul Pacheco <notifications@github.com>
Skickat: den 17 oktober 2017 15:17
Till: pernyblom/pernyblom.github.io
Kopia: Subscribed
Ämne: [pernyblom/pernyblom.github.io] Split up into different repos (#1)


This is a suggestion, each one of these projects should have their own github repository.
